dealers_styles.css

```diff
@@ -0,0 +1,124 @@
+body{
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: black;
+  margin: 40px;
+}
+
+.nav {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  background-color: rgb(2, 2, 2);
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  }
+
+.nav li {
+  flex: 1;
+  justify-content: center;
+  color: rgb(133, 130, 103);
+ }
+
+ .nav a {
+  font-size: 1.3em;
+  color: #fff;
+  text-decoration: none;
+  padding: 5px;
+  border-radius: 5px;
+}
+.nav a:hover {
+  color: #202b47;
+  background-color: #fff;
+  box-shadow: #d3d3d3;
+}
+
+.nav a {
```

Comment:
"nav a" is used two times. one in line 23 and again on line 36. Please combine these two.